build-tools-internal/src/integTest/groovy/org/elasticsearch/gradle/fixtures/AbstractGradleInternalPluginFuncTest.groovy

```diff
@@ -8,11 +8,11 @@
 
 package org.elasticsearch.gradle.fixtures
 
-import org.elasticsearch.gradle.internal.conventions.precommit.PrecommitPlugin
+import org.gradle.api.Plugin
```

Comment:
Are these intentional changes?

Reply:
Sorry about that, I accidentally merged in `master` to get a test fix, without updating the feature branch first. I'll fix this now.